BlueHarvest/ViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ViewModel.swift
+//  BlueHarvest
+//
+//  Created by David Nutter on 3/19/17.
+//  Copyright © 2017 David Nutter. All rights reserved.
+//
+
+import UIKit
+
+protocol ViewModelDelegate: class {
+    func viewModelDidUpdate(_ viewModel: ViewModel)
+}
+
+class ViewModel {
+
+    weak var delegate: ViewModelDelegate?
+    var profiles = [Profile]() // Model
+
+    private var client: APIClient!
```

Comment:
APIClient is a protocol though so I wouldn't be able to instantiate it like that. Perhaps my naming scheme is a bit confusing...